CRM/Core/BAO/SchemaHandler.php

```diff
@@ -295,7 +295,7 @@ public static function buildForeignKeySQL($params, $separator, $prefix, $tableNa
    * @return bool
    */
   public static function alterFieldSQL($params, $indexExist = FALSE, $triggerRebuild = TRUE) {
-
+    CRM_Core_Error::deprecatedFunctionWarning('function no longer in use / supported');
```

Comment:
Probably should add a `@deprecated` annotation too.

Reply:
Ok - I'll add that to the last PR (rather than invalidate tests on this)